app/workers/functions.php

```diff
@@ -507,7 +505,7 @@ public function cleanup(): void
                 $stdout = '';
                 $stderr = '';
 
-                if(Console::execute("docker stop {$first['name']}", '', $stdout, $stderr, 30) !== 0) {
+                if(Console::execute("docker rm -f {$first['name']}", '', $stdout, $stderr, 30) !== 0) {
```

Comment:
@eldadfux  we should probably stop the container before we remove it else `docker rm -f` would throw an error.

Reply:
The -f option allow us to remove containers without needing to stop them. Its just work faster than sending 2 CLI commands.

Reply:
Okay awesome 👍